src/Http/Routing/src/Builder/EndpointRouteBuilderExtensions.cs

```diff
@@ -327,9 +327,15 @@ public static class EndpointRouteBuilderExtensions
             };
 
             // Add delegate attributes as metadata
-            foreach (var attribute in requestDelegate.Method.GetCustomAttributes())
+            var attributes = requestDelegate.Method.GetCustomAttributes();
```

Comment:
Could `Method` ever be null? Is it worth `requestDelegate.Method?.GetCustomAttributes()`

Reply:
Is this exception from `Method` ever something to worry about? https://docs.microsoft.com/en-us/dotnet/api/system.delegate.method?view=netcore-2.2#exceptions

Reply:
Hmm, I'm not sure what combination of things would throw that exceptions. I'll see if I can reproduce it.